### PR TITLE
[EPMLABSBRN-77] - Added uniqe constraint and index for word and audioFileUrl resource attributes

### PR DESCRIPTION
**Pull request by RiveriRiv**

### CODE REVIEW

File: src/main/kotlin/com/epam/brn/model/Resource.kt [hunk 1]
```diff
@@ -2,13 +2,20 @@ package com.epam.brn.model
...
-import javax.persistence.ManyToMany
+import javax.persistence.GeneratedValue
 import javax.persistence.Column
+import javax.persistence.ManyToMany
+import javax.persistence.GenerationType
 
 @Entity
+@Table(
+    uniqueConstraints = [UniqueConstraint(columnNames = ["word", "audioFileUrl"])],
+    indexes = [Index(name = "word_audio_file_idx", columnList = "word, audioFileUrl")]
```
Comment (goopnigoop) [inline src/main/kotlin/com/epam/brn/model/Resource.kt]: suggestion lets add one more index only for audioFileUrl too

File: src/main/kotlin/com/epam/brn/model/Resource.kt [hunk 2]
```diff
@@ -17,9 +24,7 @@ data class Resource(
     val audioFileUrl: String,
```
Comment (ElenaSpb) [inline src/main/kotlin/com/epam/brn/model/Resource.kt]: as audioFileUrl is requered - maybe add this annotation
@Column(nullable = false)
to its too?

File: src/main/kotlin/com/epam/brn/model/Resource.kt (hunk 1 above)
Comment (ElenaSpb) [reply src/main/kotlin/com/epam/brn/model/Resource.kt]: maybe for index for word only - as we will find recoueses for words

Comment (RiveriRiv) [reply src/main/kotlin/com/epam/brn/model/Resource.kt]: done

**Pull request merged**